aws/resource_aws_iam_user.go

```diff
@@ -87,37 +98,37 @@ func resourceAwsIamUserRead(d *schema.ResourceData, meta interface{}) error {
 		UserName: aws.String(d.Id()),
 	}
 
-	getResp, err := iamconn.GetUser(request)
+	output, err := iamconn.GetUser(request)
 	if err != nil {
-		if iamerr, ok := err.(awserr.Error); ok && iamerr.Code() == "NoSuchEntity" { // XXX test me
+		if isAWSErr(err, iam.ErrCodeNoSuchEntityException, "") { // XXX test me
```

Comment:
Has this been tested? Do we still need this comment?

Reply:
Test added 👍 

```
make testacc TEST=./aws TESTARGS='-run=TestAccAWSUser_disappears'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -run=TestAccAWSUser_disappears -timeout 120m
=== RUN   TestAccAWSUser_disappears
--- PASS: TestAccAWSUser_disappears (8.90s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	9.475s
```